src/pipeline_blocks.py

```diff
@@ -189,6 +189,50 @@ def classifier_light_gbm_stacking(features, config, train_mode, suffix, **kwargs
                          **kwargs)
     return light_gbm
 
+def classifier_log_reg_stacking(features, config, train_mode, suffix, **kwargs):
+    model_name = 'log_reg{}'.format(suffix)
+
+    if train_mode:
+        features_train, features_valid = features
+        if config.random_search.log_reg.n_runs:
+            transformer = RandomSearchOptimizer(TransformerClass=get_sklearn_classifier(LogisticRegression, normalize=True, **config.log_reg),
```

Comment:
@Ninoko I'd suggest that you simply go with if n_runs: raise NotImplementederror else LOGIC